File: Arrays/python/insertionArray.py
A =[10, 20, 30, 40, 50]
x = 25
pos = 2

A.append(0) # to increase size by 1
for i in range(len(A)-2, pos-1, -1):
    A[i+1] = A[i]

A[pos] = x

print("Array after insertion:", A)

File: Arrays/javascript/InsertionArray.js
let A = [10, 20, 30, 40, 50];
let x = 25;
let pos = 2;

A.push(0);

for (let i = A.length - 2; i >=pos; i--){
    A[i +1] = A[i];
}

A[pos] =  x;

console.log("Array after insertion: ", A);
